x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshots.java

```diff
@@ -237,15 +237,11 @@ public void onIndexModule(IndexModule indexModule) {
 
     @Override
     public List<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
-        if (SEARCHABLE_SNAPSHOTS_FEATURE_ENABLED) {
```

Comment:
This change means that we register these actions whether the feature flag is enabled or not, which I think we should avoid. Can we revert this?

Reply:
I've been confused by the TransportClient for which I thought we should not force a system prop to be present to load the appropriate actions.

But the feature flag will be removed at some point, so we don't really care. I reverted this.